### ISSUE TITLE
.vscode file on github 

### ISSUE THREAD

**Issue by khadija-nur** [fac20/week-9-bfop#33]
Would you consider adding .vscode in your gitignore file to prevent it from being pushed to github(unless you want to do that of course)? It might help simplify your file structure ⭐ 📁

**Comment by azizi-a** [comment]
Good point. It might be good to have the same debug environment shared across users though. What do you think?
